### PR TITLE
Fixes #304 - remove trailing slash on j2objc.home if present

### LINKED ISSUE #304: Trailing forward slash in j2objc.home path breaks pods config
@alexmetcalfe, tracking your new issue here and for a fix.

### PR DESCRIPTION
**Pull request by advayDev1**
TESTED=yes

@alexmetcalfe cc

### CODE REVIEW

File: src/test/groovy/com/github/j2objccontrib/j2objcgradle/tasks/UtilsTest.groovy [hunk 1]
```diff
@@ -102,6 +102,18 @@ class UtilsTest {
         assert j2objcHomeWritten.equals(j2objcHomeRead)
     }
 
+    @Test
+    public void testJ2objcHome_LocalPropertiesWithSlash() {
```
Comment (brunobowden) [inline src/test/groovy/com/github/j2objccontrib/j2objcgradle/tasks/UtilsTest.groovy]: WithSlash => WithTrailingSlash

File: src/test/groovy/com/github/j2objccontrib/j2objcgradle/tasks/UtilsTest.groovy [hunk 2]
```diff
@@ -102,6 +102,18 @@ class UtilsTest {
...
+        // Write j2objc path to local.properties file within the project
+        String j2objcHomePath = File.createTempDir('J2OBJC_HOME', '').path
+        String j2objcHomeWritten = j2objcHomePath + '/'
```
Comment (brunobowden) [inline src/test/groovy/com/github/j2objccontrib/j2objcgradle/tasks/UtilsTest.groovy]: j2objcHomeWritten => j2objcHomePathWithSlash

**Comment by brunobowden** [comment]
LGTM - no strong feeling about my comments

**Comment by advayDev1** [comment]
done and done

**Pull request merged**